### DISCUSSION TITLE
Marine

### DISCUSSION THREAD

**Discussion by roly151** [trickypr/bom-weather-docs#8] [Q&A] [answered]
Hi, 

Have you found any Marine forecast endpoints? I can see that the location information indicates if it has a marine forecast, but would we need to then get the forecast from the ftp server, or is it available through this api?

**Comment by roly151** [comment, marked as answer]
Found it - for those who might be interested. Marine can be found through using the marine_area_id and tidal_point from the location information endpoint (Example: https://api.weather.bom.gov.au/v1/locations/r1ppnx )

The two endpoints for Marine are then:
/v1/locations/marine/r1ppnx/forecasts/hourly
/v1/locations/marine/<marine_area_id>/forecasts/daily
/v1/locations/marine/<tidal_point>/forecasts/tide

**Reply by trickypr** [reply]
Thanks for providing this information. I will look into adding this to the docs when I get some free time.
